### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by MaamuT** [aquasarus/Starfield-Interactive-Smart-Slate#13] [Ideas]
Possibility of deleting an incorrect entry, if I make a mistake by adding an animal or floral species, or an outpost, it would be nice if I could delete it.

Ability to sort the system list by name.

Possibility of folding/unfolding the system to save space.

A global search engine allowing you to search for materials, resources or outposts by name.

**Comment by aquasarus** [comment]
Thanks for your suggestions! Here are my thoughts:
- The ability to delete outposts is there! Just right-click on the outpost and choose Delete. Hover over the 🛈 icons in the app to see these tips.
- I decided not to add deleting fauna/flora because the nature of the surveying game loop is one-directional. As you progress, you will never "unsurvey" a lifeform, so to speak. The easiest way to correct an error is simply to scan another fauna/flora, and then rename the mistake to a real lifeform :).
- Sorting and collapsing are both good ideas! For now, I thought filters (Lifeforms and Outposts) would be the best addition to help you find the solar system you want. Once I finish adding more core features, I will look into this for sure!
- Could you elaborate on this global search engine idea? There is no outpost search at the moment, but you can certainly search for both organic and inorganic resources in their respective tabs.

**Comment by MaamuT** [comment]
Ok, I was wrong about the outposts.

I understand about the fauna/flora, no problem.

Okay for the research, it's true that it's already good as it is, I withdraw ;)
